iModelCore/ecobjects/PublicApi/ECObjects/ECSchema.h

```diff
@@ -2929,6 +2929,19 @@ struct SearchPathSchemaFileLocater : IECSchemaLocater, RefCountedBase, NonCopyab
     ECOBJECTS_EXPORT static SearchPathSchemaFileLocaterPtr CreateSearchPathSchemaFileLocater(bvector<WString> const& searchPaths, bool includeFilesWithNoVerExt=false);
 };
 
+//=======================================================================================
+//! Reads schemas from a given set of Xml strings
+//=======================================================================================
+struct EXPORT_VTABLE_ATTRIBUTE StringSchemaLocater : IECSchemaLocater, NonCopyableClass

```

Comment:
Now that this is here we should remove the one in TestFixture.h and use this one instead

Reply:
Removed StringSchemaLocater from TestFixture.h